tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Common/DocumentProvision.cs

```diff
@@ -497,6 +497,14 @@ public GenericResponseVM PerformContentCheck(string clientUrl, string folderUrl,
                                 configuration.GetSection("Search").GetSection("SearchColumnsUIPickerForDocument").GetSection("docId").Key,
                                 searchResult[key].ToString());
                         }
+

```

Comment:
For project center, in document search, practice group is shown. So added this new property in search json results